solarforecastarbiter/io/reference_observations/common.py

```diff
@@ -387,6 +393,11 @@ def create_forecasts(api, site, variables):
     variables : list-like
         List of variables to make a new forecast for each of TEMPLATE_FORECASTS
     """
+    if not is_in_nwp_domain(site):
+        logger.warning(
+            'Site %s is outside the domain of the current NWP forecasts',
+            site.name)
+        return []
```

Comment:
perhaps more pythonic to raise an exception here and let the calling code decide what to do. no objection to keeping this if that's too complicated for now.